docs/extend/install-configure-zos-extensions.md

```diff
@@ -0,0 +1,144 @@
+# Install and Configure Zowe Server Component
+
+## Install with zowe-install-component.sh <Badge text="Technical Preview"/>
+
+_Note: this section is for technical preview and we are happy to hear any feedbacks. Content in this section may be changed or improved in the future._
+
+_Note: this feature is added with Zowe v1.19.0 release._
+
+From v1.19.0, Zowe ships a `bin/zowe-install-components.sh` tool to help you install any Zowe server component (extension). Zowe core components are also installed with this tool. To make the tool works better, the component to be installed is suggested to follow [Zowe server component package format standard](packaging-zos-extensions.md#zowe-server-component-package-format).
+
+The tool can be executed from z/OS USS, and it takes these command line parameters:
+
+- **`-o|--component-file`**: This is required. It's the path to the component package or directory.
+- **`-c|--component-name`**: This is optional. It's the name of the component. If `NODE_HOME` is defined, and the component has manifest file, this value is optional. Otherwise it's required.
+- **`-i|--instance-dir`**: This is optional. It's the path to Zowe instance directory. If this has a value, the script will also execute `bin/zowe-configure-component.sh` once it's installed.
+- **`-d|--target-dir`**: This is optional. It's the path to installation target directory. For native component, default value is `${ZOWE_ROOT_DIR}/components`. For non-native component, the script will check `ZWE_EXTENSION_DIR` variable value. If it's also empty, the script will fall back to default target directory, which is `/global/zowe/extensions`.
+- **`-e|--auto-encoding`**: This is optional. It defines whether we should automatically tag the encoding of the files shipped with the component. Default value is `auto`, which means the script will determine if the automatic tagging is needed or not. This option is only applicable for z/OS. Allowed values are:
+  * `yes`: automatically tag the encoding of the files,
+  * `no`: do not automatically tag encoding of the files,
+  * `auto`: only tag when manifest is in `ISO8859-1` encoding.
+- **`-n|--native`**: This is an optional boolean. It defines whether this component is bundled into Zowe package as core component.
```

Comment:
Again, I think the term 'native' is misleading. Native to me means written for a specific platform. I wonder if this should even be doc'd for extenders as they aren't going to be core components. Same below as well